homeassistant/components/hausbus/gateway.py

```diff
@@ -0,0 +1,185 @@
+"""Representation of a Haus-Bus gateway."""
+
+import asyncio
+from collections.abc import Callable, Coroutine
+from typing import Any, cast
+
+from pyhausbus.ABusFeature import ABusFeature
+from pyhausbus.BusDataMessage import BusDataMessage
+from pyhausbus.de.hausbus.homeassistant.proxy.Controller import Controller
+from pyhausbus.de.hausbus.homeassistant.proxy.controller.data.Configuration import (
+    Configuration,
+)
+from pyhausbus.de.hausbus.homeassistant.proxy.controller.data.ModuleId import ModuleId
+from pyhausbus.de.hausbus.homeassistant.proxy.controller.data.RemoteObjects import (
+    RemoteObjects,
+)
+from pyhausbus.HomeServer import HomeServer
+from pyhausbus.IBusDataListener import IBusDataListener
+from pyhausbus.ObjectId import ObjectId
+
+from homeassistant.components.light import DOMAIN as LIGHT_DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .device import HausbusDevice
+from .entity import HausbusEntity
+from .event_handler import IEventHandler
+from .light import (
+    Dimmer,
+    HausbusDimmerLight,
+    HausbusLedLight,
+    HausbusLight,
+    HausbusRGBDimmerLight,
+    Led,
+    RGBDimmer,
+)
+
+
+class HausbusGateway(IBusDataListener, IEventHandler):  # type: ignore[misc]
+    """Manages a single Haus-Bus gateway."""
+
+    def __init__(self, hass: HomeAssistant, config_entry: ConfigEntry) -> None:
+        """Initialize the system."""
+        self.hass = hass
+        self.config_entry = config_entry
+        self.bridge_id = "1"
```

Comment:
Why is this hard coded like this? If bridge ID is always 1, maybe just remove it from the class?

Reply:
I thought I could make use of the bridge_id, but it turned out obsolete.